video/out/wayland_common.c

```diff
@@ -894,9 +894,17 @@ static void handle_toplevel_config(void *data, struct xdg_toplevel *toplevel,
     wl->toplevel_width = width;
     wl->toplevel_height = height;
 
-    /* Don't do anything here if we haven't finished setting geometry. */
-    if (mp_rect_w(wl->geometry) == 0 || mp_rect_h(wl->geometry) == 0)
+    if ((mp_rect_w(wl->geometry) == 0 || mp_rect_h(wl->geometry) == 0)) {
```

Comment:
nit, but those additional parens are not needed, no?

Reply:
Not sure where those crept in from.